packages/cms-cli/commands/auth.js

```diff
@@ -29,19 +28,15 @@ const {
 } = require('../lib/prompts');
 const {
   addConfigOptions,
-  addLoggerOptions,
```

Comment:
Guessing this and `addHelpUsageTracking` are being done at a higher level now?

Reply:
Yep they are both added in cli.js